databroker/src/broker.rs

```diff
@@ -157,6 +160,7 @@ pub enum QueryError {
 pub enum SubscriptionError {
     NotFound,
     InvalidInput,
+    InvalidSubscibeBufferSize,
```

Comment:
There's a typo (Subscibe instead of Subscribe).

But might as well remove it completely and have `InvalidBufferSize` instead as "subscribe" implied by the fact it's a subscription error.

Reply:
Updated